elastalert/kibana_external_url_formatter.py

```diff
@@ -157,3 +157,15 @@ def create_kibana_external_url_formatter(
         return ShortKibanaExternalUrlFormatter(base_url, auth, security_tenant, new_shortener, verify)
 
     return AbsoluteKibanaExternalUrlFormatter(base_url, security_tenant)
+    
+def create_opensearch_external_url_formatter(
```

Comment:
Why is this in the kibana_external_url_formatter.py file, and why does it reference `kibana_url` on line 168?

Reply:
Since I only used one function, I found it easier to put it there in an already existing file with existing assets I could reuse

Reply:
I moved it to a new python file opensearch_external_url_formatter.py and created a test file for it